Formula/gst-plugins-bad.rb

```diff
@@ -38,14 +38,14 @@ class GstPluginsBad < Formula
   depends_on "rtmpdump" => :optional
   depends_on "schroedinger" => :optional
   depends_on "sound-touch" => :optional
-  depends_on "srtp" => :optional
+  depends_on "srtp@1.5" => :optional
   depends_on "libvo-aacenc" => :optional
 
   def install
     args = %W[
       --prefix=#{prefix}
       --disable-yadif
-      --disable-sdl
```

Comment:
quoting the explanation in my commit message:

> --disable-sdl is no longer a configure option.

> But with --disable-examples, it will not check for sdl, which is only used for the examples.